araproc/analysis/standard_reco.py

```diff
@@ -206,27 +250,21 @@ def do_standard_reco(self, waveform_bundle):
 
         reco_results = {}
 
-        # set up the waveform map the way AraRoot wants it
-        # as a std::map<int, TGraph*>
-        waveform_map = ROOT.std.map("int", "TGraph*")()
-        ROOT.SetOwnership(waveform_map, True)
-        for chan_i in waveform_bundle.keys():
-            waveform_map[chan_i] = waveform_bundle[chan_i]
-                
-        # get the correlation functions
-        corr_functions_v = self.rtc_wrapper.correlators["distant"].GetCorrFunctions(
-                                                                    self.pairs_v,
-                                                                    waveform_map
-                                                                    )
-        corr_functions_h = self.rtc_wrapper.correlators["distant"].GetCorrFunctions(
-                                                                    self.pairs_h,
-                                                                    waveform_map
-                                                                    )
+        event_number = wavepacket["event"]
+        waveform_bundle = wavepackt["waveforms"]
```

Comment:
Typo: `wavepackt`. I'm surprised this even ran?

Reply:
Merp maybe I forgot to rerun it after I introduced that one. It's definitely running fine now.